templates/search/filter/active-filters.php

```diff
@@ -22,9 +22,11 @@
     'removeAllUrl' => $removeAllUrl,
 ] = $data;
 
+$class = !$activeFilters ? ' hidden' : '';
```

Comment:
Minor suggestion as to where to place the whitespace. I prefer more "beautiful" templates over the look of the rendered HTML. 😉

```suggestion
$class = !$activeFilters ? 'hidden' : '';
```